sdk/pom.xml

```diff
@@ -10,8 +10,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>11</source>
-                    <target>11</target>
+                    <source>15</source>
```

Comment:
I think we want to keep this at 11 if possible until some downstream projects can upgrade (gateway)

Reply:
The build is breaking b/c of this

Reply:
will fix it.